styles.css

```diff
@@ -69,13 +70,19 @@ h3 {
     margin: 0;
 }
 
-article, main {
+article, main, .back-button {
     margin: 40px auto;
     background-color: #FFFFFF;
     padding: 15px;
     border-radius: 5px;
 }
 
+.back-button {
+    width: -webkit-fit-content; width: -moz-fit-content; width: fit-content;
```

Comment:
These should be on seperate lines, sorry they were in a line because of pasting into github comments